### ISSUE TITLE
improve startup time of application.

### ISSUE THREAD

**Issue by neutrinog** [unfoldingWord-dev/ts-desktop#183]
The application needs to be able to start and display an interface to the user in under 2 seconds. If we need time to load things we should display a splash screen "Welcome to translationStudio" with some sort of progress indicator (at least a spinning wheel).

**Comment by vleong2332** [comment]
Btw, I'm taking a shot at this.

**Comment by vleong2332** [comment]
Just doing a quick update here: I was doing some timing. Basically, for windows, the app opens in about 20 seconds mark, which is abysmal. In Linux VM it only takes about 7.5 seconds.

I came across these conversations that are relevant to our issue:
https://github.com/nwjs/nw.js/issues/811 - which is a duplicate of a longer issue that spans for more than 3 years!: https://github.com/nwjs/nw.js/issues/151#issuecomment-65222370

From what I got, it seems people are having the same problem of opening the app in windows.

Sadly, I haven't found any solution yet. But I thought of posting it to see maybe some of you have a brilliant solution.
